lib/ansible/modules/packaging/os/apt.py

```diff
@@ -124,6 +126,16 @@
     default: no
     choices: [ "yes", "no" ]
     version_added: "2.4"
+  autostart:
+    description:
+      - If C(yes), the module creates the C(/usr/sbin/policy-rc.d) file which prevents automatic service startup if the
+        package contains one.
+      - If C(no), the module will remove the C(/usr/sbin/policy-rc.d) file which will cause automatic service startup
+        if the package contains one.
+      - If not set, the module won't remove the C(/usr/sbin/policy-rc.d) file if it exists.
```

Comment:
I'm unsure if we should create and leave this file for other people attempting to install then being surprised by their services not being created.

I'm tempted to just document this workaround and let users create/remove the file via another task.

Reply:
I understand. But as the behavior is documented and the file would be created only when the user explicitly enables it, the surprises would be reduced to minimum. Anyway, only badly designed roles do not have tasks to manage the service.

But we could also change the behavior to support only `no` and `null` values. If `no` is set and the file doesn't exist, the file would be created at the beginning of the task and removed a the end of it. If set to `null` (default) nothing would happen. What do you think?

Reply:
if module is going to create this i would say that it should remove it in the end, but only if it created it during this run

Reply:
OK. I will modify the PR to do that.